src/components/BackToTopBtn.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <q-btn
+    v-back-to-top
+    round
+    class="fixed-bottom-right"
+    style="margin: 0 15px 15px 0; background-color: #4a3520"
```

Comment:
Can you move these styles into a scoped class?

Also, I think it's best if you use one of our theme colors, for example by adding `bg-primary` as class.
If none of the theme colors fits, we have to rework our theme ;)